Sources/SDL+JNIExtensions.swift

```diff
@@ -8,10 +8,10 @@
 
 import JNI
 
-@_silgen_name("Android_JNI_GetActivityClass")
+@_cdecl("Android_JNI_GetActivityClass")
 public func getSDLViewClass() -> JavaClass
 
-@_silgen_name("SDL_AndroidGetActivity")
+@_cdecl("SDL_AndroidGetActivity")
```

Comment:
same here

Reply:
Hmm ok this actually makes sense because we’re not defining anything about the function with an empty declaration. It’s just signalling that the function exists.